### PR TITLE
Ajout de mon profile : Nicolas Verinaud

### PR DESCRIPTION
**Pull request by nverinaud**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -16395,6 +16395,59 @@ var data = {
         "twitter": "saby_nastasia",
         "mail": "nastasia.saby@gmail.com"
       }
+    },
+    {
+      "since": "2017-12-21",
+      "name": "Nicolas Verinaud",
+      "bio": "👨‍💻 Artisan Logiciel chez Ryfacto",
```
Comment (dadoonet) [inline baggers.js]: Ha! let see what it will do when rendering... :p

**Review by dadoonet — APPROVED** [review]
LGTM. Could you rebase it so we can merge?

**Comment by nverinaud** [comment]
Done 😉

**Comment by dadoonet** [comment]
Something is weird. It sounds like you are also adding #257 again...

Could you just rebase your very first commit on master?

**Comment by nverinaud** [comment]
Sorry I messed up a little bit. This one seems better. 😉

**Review by dadoonet — APPROVED** [review]

**Comment by dadoonet** [comment]
Awesome! Merged. Welcome!

**Comment by nverinaud** [comment]
Yeah ! Thanks ! 🎉

**Pull request merged**